### DISCUSSION TITLE
Are there expected to be any new 0.7.x releases?

### DISCUSSION THREAD

**Discussion by liss-h** [tokio-rs/axum#2907] [Q&A] [answered]
Hi 🙂, 
sorry I don't mean to annoy you and hope you don't get this question way too often.
We were wondering if we can expect any new 0.7.x releases or if the next release is going to be 0.8. I'm asking because there are some unreleased features on main that we would like to use and it seems like 0.8 still has a long way to go. If there will be no more 0.7.x releases, is there any commit on main you consider "stable" (so that it can be used almost like a release)?

**Comment by jplatte** [comment, marked as answer]
Yes, there's a plan for a point release to axum-core and axum, though not axum-extra (let me if you think there should be one).

I'm hoping I can get those releases published next week, but I can't promise anything since I don't have publish permissions myself currently.

Out of curiosity, which change(s) are you interested in?

**Reply by liss-h** [reply]
Thx, good to know.
We are mostly interested in `Serve::tcp_nodelay` & `Serve::local_addr`.

**Reply by jplatte** [reply]
The releases are out! :)
